_posts/2024-03-06-bitcoin-inquisition-#39.md

```diff
@@ -6,34 +6,57 @@ link: https://github.com/bitcoin-inquisition/bitcoin/pull/39
 permalink: /bitcoin-inquisition-39
 authors: [0xBEEFCAF3, EthanHeilman]
 components: ["consensus"]
-host: EthanHeilman
+host: EthanHeilman, arminsdev aka 0xBEEFCAF3
```

Comment:
feel free to just change this to your own handle if you're hosting. the website has an index by host

Reply:
I'll be attending and co-hosting with Ethan. Happy to leave Ethan's name up.